subprojects/logging/src/test/groovy/org/gradle/internal/deprecation/LoggingDeprecatedFeatureHandlerTest.groovy

```diff
@@ -70,8 +70,8 @@ class LoggingDeprecatedFeatureHandlerTest extends Specification {
         events.size() == 2
 
         and:
-        events[0].message == 'feature1 removal'
-        events[1].message == 'feature2 removal'
+        events[0].message.startsWith TextUtil.toPlatformLineSeparators('feature1 removal\n')
```

Comment:
This might be similar to `DeprecationMessagesTest`, we might need a better adaptation of these tests too, so that we know that we aren't sweeping problems under the carpet.